R/utils.R

```diff
@@ -107,3 +105,90 @@ print.palette <- function(x, ...) {
   rect(0, 0.9, n + 1, 1.1, col = rgb(1, 1, 1, 0.8), border = NA)
   text((n + 1) / 2, 1, labels = attr(x, "name"), cex = 1, family = "serif")
 }
+
+#' Calculate biomass from DBH data in fd_inventory() using equations from
+#' Table  . Parameters a and b and diameter at breast height (DBH, cm) range for the aboveground biomass
+#' sans leaves equation Mass = a * DBH^b for tree species occurring in the UMBS-Flux footprint.
+#'
+#' Studies used to augment Cooper’s raw data for a species are denoted by superscripts.
+#' BIOMASS BACKGROUND INFO
+#' Species	a	b	DBH range (cm)
+#' Acer rubrum a	0.0312	2.7780	1 – 40
+#' Acer saccharum b	0.1693	2.3436	3 – 66
+#' Amelanchier spp.c	0.1630	2.4940	0.3-2.7h
+#' Betula papyrifera d	0.0301	2.8387	2 – 49
+#' Fagus grandifolia b	0.1892	2.3097	3 – 66
+#' Pinus resinosa e	0.0526	2.5258	2 – 32
+#' Pinus strobus e	0.0408	2.5735	1 – 32
+#' Populus grandidentata f	0.1387	2.3498	1 – 37
+#' Populus tremuloides b	0.0589	2.6235	3 – 51
+#' Quercus rubra g	0.0398	2.7734	1 – 44
+#' a Crow and Erdmann, 1983, Perala and Alban, 1994, Young et al., 1980.
+#' b Young et al., 1980 equations used exclusively, DBH range estimated by Ker-Mikaelian and Korzukhin, 1997.
+#' c Perala and Alban, 1994 equations used exclusively.
+#' d Perala and Alban, 1994, Schmitt and Grigal, 1981, Young et al., 1980.
+#' e Ker, 1980, Perala and Alban, 1994, Young et al., 1980.
+#' f Koerper, 1977  raw data collected near UMBS-Flux tower.
+#' g Hocker and Early, 1983, Perala and Alban, 1994, Wiant et al., 1977.
+#' h Diameter at 15 cm height.
+#'
+#'
+#' @return A data frame of biomass at the individual level build from fd_inventory
+#' @export
+#' @examples
+#' fortedata::calc_biomass()
+
+calc_biomass <- function(){
+# The allometries
+allo.df <- read_csv_file("fd_biomass_allometries.csv") #this has the same equations AmeriFlux uses
+df <- fortedata::fd_inventory()
```

Comment:
```suggestion
df <- fd_inventory()
```

Otherwise, I think `devtools::load_all()`, `devtools::test()`, and friends might fall back on the installed version, rather than the development version...though I'm not 100% sure.